FluidNC/src/Spindles/Spindle.cpp

```diff
@@ -108,6 +120,43 @@ namespace Spindles {
         _speeds.push_back({ max, 100.0f });
     }
 
+    const char* Spindle::atc_info() {  // this can be used in the startup response
+        std::string atc_info = "";
+        if (_atc != NULL) {
+            atc_info = " ATC:" + _atc_name;
+        } else if (!_m6_macro._gcode.empty()) {
+            atc_info = " m6_macro:";
+        }
+        return atc_info.c_str();
+    }
+
+    // pre_select is generally ignored except for machines that need to get a tool ready
+    // set_tool is just used to tell the atc what is already installed.
+    bool Spindle::tool_change(uint32_t tool_number, bool pre_select, bool set_tool) {
```

Comment:
Perhaps call it "current_tool" instead of "set_tool".  "set" as a prefix is usually used as a verb, as in set_ and get_

Reply:
`bool set_tool` means it is coming from a stand alone T<num> rather than an M6 T<num>

Maybe `select_tool` is better?

https://linuxcnc.org/docs/2.8/html/gcode/other-code.html#sec:select-tool